packages/build-info/src/frameworks/framework.ts

```diff
@@ -132,14 +132,14 @@ export function filterByRelevance(detected: DetectedFramework[]) {
  * 3. an npm dependency was specified but matched over the config file (least accurate)
  */
 export function sortFrameworksBasedOnAccuracy(a: DetectedFramework, b: DetectedFramework): number {
-  let sort = a.detected.accuracy > b.detected.accuracy ? -1 : a.detected.accuracy < b.detected.accuracy ? 1 : 0
+  const sort = b.detected.accuracy - a.detected.accuracy
```

Comment:
just noting here in case my check was wrong

I was pretty sure that comparator function should return one of `-1 | 0 | 1` and not just any number, but it seems like my understanding was wrong - both MDN ( https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#comparefn ) and ecmascript ( https://tc39.es/ecma262/multipage/indexed-collections.html#sec-array.prototype.sort ) mention negative/positive instead of -1 / 1 so it looks like the code as-is would be fine and I must have confused it with some other language that might be more strict (?)